make_level.py

```diff
@@ -0,0 +1,117 @@
+import sys
+import os
+
+cwd = os.getcwd()
+
+def write_init(skill_name, skill_path, level_name):
+    # If skills/skill/__init__.py doesn't exist, create a basic version (no levels)
+    if not os.path.exists(os.path.join(skill_path,"__init__.py")):
+        with open(os.path.join(skill_path,"__init__.py"), 'w+') as fp:
+            level_setup = "\n".join([
+                "import pkg_resources\n",
+                "from gitgud.skills.util import BasicLevel",
+                "from gitgud.skills.util import Skill\n"
+                "skill = Skill(",
+                "    '{}',".format(skill_name),
+                "    [",
+                "    ]",
+                ")\n",
+                ])
+            fp.write(level_setup)
+    
+        
+        # Read file
+        with open(os.path.join(os.path.join("gitgud","skills"),"__init__.py"), 'r') as fp:
```

Comment:
I think you can do 
```
os.path.join("gitgud", "skills", "__init__.py")
```

Reply:
Lol yeah I used this format in other parts of the code haha idk why I started nesting them along the way. Fixed